docs/src/upgrade/release-3.8.x.asciidoc

```diff
@@ -30,6 +30,53 @@ complete list of all the modifications that are part of this release.
 
 === Upgrading for Users
 
+==== Number Conversion Step
+
+We have introduced a number conversion step, `asNumber()`, which converts the incoming traverser to the nearest parsable type if no argument is provided, or to the desired numerical type, based on the number token (`N`) provided.
```

Comment:
the description here is a a copy of what's in the docs. upgrade documentation is should do more to announce a feature and why it's useful. what can be done with this step that you couldn't do before? how does this step make something easier? upgrade docs should describe that sort of thing.